src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer: Bulldozer started working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer:Bulldozer stopped working");

```

Comment:
The message in the `stopWork` method is not properly formatted. There should be a space after the colon to make the message more readable. Consider changing it to `System.out.println("Bulldozer: Bulldozer stopped working");` for consistency with the `doWork` method.